internal/operations/manager.go

```diff
@@ -134,11 +159,19 @@ func (om *operationsManager) RunOperation(ctx context.Context, op *core.Prepared
 			ErrorMessage:   err.Error(),
 			Output:         outputs,
 		})
-	} else if complete {
+	} else {
+		// No error so move us from "Initialized" to "Pending"
+		newState := core.OpStatusPending
```

Comment:
This is introducing a race condition - see https://github.com/hyperledger/firefly/issues/1256